react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,24 @@
+import { isArray } from './utils';
+
+export function min(...args) {
+  if(args.length && isArray(...args)){
```

Comment:
You can use the first entry of the array since you're not handling/receiving a second array as parameter. Once you do that, you can return the `Math.min` results without creating a new `const` beforehand.